src/apis/stockx/user/index.js

```diff
@@ -0,0 +1,70 @@
+import Api from '../../base';
+
+import { getState, submitCredentials, checkStatus } from './helpers';
+
+export default class UserApi extends Api {
+  // todo:
```

Comment:
Will open a follow up issue to implement the remainder of methods. Doesn't necessarily need to be included in this PR.

Reply:
Tracked in https://github.com/matthew1232/stockx-api/issues/6